Source/WebCore/dom/SlotAssignment.cpp

```diff
@@ -453,36 +457,45 @@ void ManualSlotAssignment::renameSlotElement(HTMLSlotElement&, const AtomString&
 {
 }
 
-void ManualSlotAssignment::addSlotElementByName(const AtomString&, HTMLSlotElement&, ShadowRoot&)
+void ManualSlotAssignment::addSlotElementByName(const AtomString&, HTMLSlotElement& slot, ShadowRoot& shadowRoot)
 {
+    if (!m_slotElementCount)
+        shadowRoot.host()->setHasShadowRootContainingSlots(true);
+    ++m_slotElementCount;
+    ++m_slottableVersion;
+    if (auto* nowAssgined = assignedNodesForSlot(slot, shadowRoot))
```

Comment:
Typo: `nowAssgined`. Also we may not need this local variable at all.

Reply:
Oh yeah, I'm gonna get rid of the variable.